test/src/test/resources/fixtures/skip-include-fields/handles-include-directive-on-single-field-whose-parent-returns-null.yml

```diff
@@ -0,0 +1,60 @@
+name: handles include directive on single field whose parent returns null
+enabled:
+  current: false # Not fixing
+  nextgen: true
+overallSchema:
+  service: |
+    type Query {
+      foo: Foo
+    }
+    type Foo {
+      id: String
+    }
+underlyingSchema:
+  service: |
+    type Query {
+      foo: Foo
+    }
+    type Foo {
+      id: String
+    }
+query: |
+  query ($test: Boolean!) {
+    foo {
```

Comment:
do we have a test for when a parent field `@skip` resolves to `true`, and a child field `@skip` resolves to `false`? This should result in the parent being removed

Reply:
Ok done in https://github.com/atlassian-labs/nadel/commit/eac63b3e7fcba68e30c97daba5035d21f11019f3

Also added test for with default values for the variables.